src/components/CardTypeSection/CardTypeSection.tsx

```
@@ -0,0 +1,18 @@
+import styles from './CardTypeSection.module.css';
+import { InputSection } from '../InputSection/InputSection';
+
+type Props = {};
+
+export default function CardTypeSection({}: Props) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)